server/engine/src/main/java/org/eclipse/lsp/cobol/core/engine/analysis/EmbeddedCodeListener.java

```diff
@@ -209,10 +175,6 @@ private void configureParser(Parser parser) {
    * @return a visitor
    */
   public ParseTreeVisitor<List<Node>> instanceVisitor(Position position, EmbeddedLanguage lang) {
-    if (EmbeddedLanguage.CICS == lang) {
-      return new CICSVisitor(position, programUri, extendedDocument);
-    }
-
     if (EmbeddedLanguage.SQL == lang) {
```

Comment:
Jut to get the picture: Do you plan to later move out DB2, too (giving the option to use other Dialects like Oracle later and giving the option to run the parser with less need for time/resource with not "registering" any EXEC SQL?

Reply:
Yes, that the eventual plan

Reply:
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2098